src/main/java/uk/gov/hmcts/sscs/service/CcdService.java

```diff
@@ -110,18 +89,21 @@ public String caseToken() throws CcdException {
         return token;
     }
 
-    public ObjectNode generateResponse(String appealNumber, String surname) {
-        //TODO: Check surname is valid for appeal number by checking CCD and update tests
+    public CcdCase findCcdCaseByAppealNumber(String appealNumber) throws CcdException {
 
-        return TrackYourAppealJsonBuilder.buildTrackYourAppealJson(
-                findCcdCaseByAppealNumber(appealNumber));
-    }
-
-    public CcdCase findCcdCaseByAppealNumber(String appealNumber) {
-
-        //TODO: Placeholder, implement this properly as part of future story
-        //TODO: Possibly remove this method and call the new DAO class directly?
-        return new CcdCase();
+        ResponseEntity<CcdCase> responseEntity = null;
+        try {
+            serviceToken = authClient.sendRequest("lease", POST, "");
+            userToken = "Bearer " + idamClient.post("testing-support/lease");
```

Comment:
Guessing this part is ok whilst we are using our test environment but we might need to come up with a generic way of generating the user token across our environments